### DISCUSSION TITLE
Raspberry Pi 4 - HM-MOD-RPI-PCB - Keine Gerätekommunikation

### DISCUSSION THREAD

**Discussion by marwal87** [OpenCCU/OpenCCU#1243] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen.

Ich kann jedes Gerät ohne Probleme in die aktuelle Version einbinden/anlernen, nachdem die Geräte aber eingebunden sind, bekomme ich nur Störungen. Jegliche Aktionen werden nicht mehr erkannt und es kann keine Verbindung mehr zum Sensor hergestellt werden.
Betreibe ich die gleichen HM-MOD-RPI-PCB an einem RPI3B+ funktioniert alles wunderbar.

Hardware:
2x HM-MOD-RPI-PCB
2x RPI4-2GB
2x original Netzteil
2x SanDisk 32GB
(es wird nur 1x benötigt, habe es aber mit diversen Konstellationen versucht, um weitestgehend ausschließen zu können, dass einer der Komponenten defekt ist - deswegen ist alles doppelt vorhanden)

Was habe ich bisher unternommen:
- anderer RPI4
- anderer HM-MOD-RPI-PCB
- andere der SD-Karte
- anderes Netzteil
- andere Infrastruktur -> Gebäudewechsel

Bin für jeden Ratschlag dankbar! Besten Dank im Voraus!

**Comment by jens-maus** [comment, marked as answer]
Der RaspberryPi4 ist leider dafür bekannt eine recht große RF-Störwolke um sich herum zu haben und ist daher eher suboptimal für den direkten Betrieb eines Funkmodules auf dem GPIO Bus geeignet.

**Reply by marwal87** [reply]
Besten Dank für die zügige Antwort - dann belasse ich es beim 3er.
